app/src/pages/authoring/[resourceType]/[id].tsx

```diff
@@ -226,29 +229,38 @@ export default function ResourceAuthoringPage() {
                 data={libOptions}
               />
             )}
-            <Button
-              w={120}
-              onClick={() => {
-                const [additions, deletions] = parseUpdate(
-                  url,
-                  identifierValue,
-                  identifierSystem,
-                  name,
-                  title,
-                  description,
-                  library
-                );
-                resourceUpdate.mutate({
-                  resourceType: resourceType as ArtifactResourceType,
-                  id: id as string,
-                  additions: additions,
-                  deletions: deletions
-                });
-              }}
-              disabled={!isChanged()} // only enable the submit button when a field has changed
-            >
-              Submit
-            </Button>
+            <Group>
+              <Button
+                w={120}
+                onClick={() => {
+                  const [additions, deletions] = parseUpdate(
+                    url,
+                    identifierValue,
+                    identifierSystem,
+                    name,
+                    title,
+                    description,
+                    library
+                  );
+                  resourceUpdate.mutate({
+                    resourceType: resourceType as ArtifactResourceType,
+                    id: id as string,
+                    additions: additions,
+                    deletions: deletions
+                  });
+                }}
+                disabled={!isChanged()} // only enable the submit button when a field has changed
+              >
+                Update
+              </Button>
+              <Button
+                w={120}
+                onClick={() => setIsModalOpen(true)}
+                //TODO/question: disabled={} any disable needed? any necessary fields?
+              >
+                Release
```

Comment:
Ah yes, I also noticed this! I think we should make a task for this. However, in the meantime, do you think it would be useful to at least have some sort of indication that changes have not been saved? So the user knows to update before releasing? Or should we just wait to do this task?